src/main/resources/api-specs/user-openapi.yaml

```diff
@@ -17,12 +17,12 @@ tags:
     description: "Operations related to user profiles"
 
 paths:
-  /api/v1/users/{userId}:
+  /api/v1/users/:
     get:
       tags:
         - "User"
       summary: "Get an user by its ID"
-      description: "Get a detailed user profile for the given user ID."
+      description: "Retrieve a detailed user profile for the given user ID."
```

Comment:
why did you remove this id from path in url? The method name and desc are kinda confusing

Reply:
Fixed